code/game/objects/items/storage/firstaid.dm

```diff
@@ -517,3 +517,111 @@
 /obj/item/storage/pill_bottle/paxpsych/PopulateContents()
 	for(var/i in 1 to 5)
 		new /obj/item/reagent_containers/pill/paxpsych(src)
+
+/obj/item/storage/organbox
+	name = "organ transport box"
+	desc = "An advanced box with an cooling mechanism that uses cryostylane or other cold reagents to keep the organs or bodyparts inside preserved."
+	icon_state = "organbox"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	throw_speed = 3
+	throw_range = 7
+	custom_premium_price = 1100

```

Comment:
It costs 1100 credits, and doesn't come with free ice? (Where do you get ice from anyway, the Bar dispenser/vemdor, right?)

Reply:
cyrostylane is a chem made in chemistry (which docs have access ) and if you happen to be without chem access for some reason you can get ice from barman or hacked drink vending machines